core/src/main/scripts/importer/metaImport.py

```diff
@@ -52,6 +52,10 @@ def interface():
                                    help='Path to a directory of cBioPortal '
                                         'info files to be used instead of '
                                         'contacting the web API')
+    portal_mode_group.add_argument('-n', '--no_portal_checks', default=False,
+                                       action='store_true',
+                                       help='Skip tests requiring information '
+                                            'from the cBioPortal installation')
```

Comment:
I would change the description here to `hep='start the loading process with minimal validation, skipping tests that require information from the cBioPortal installation. Please note that the loading process is not guaranteed to succeed in this case'`